CMakeLists.txt

```diff
@@ -471,3 +471,8 @@ install(FILES
   "${PROJECT_BINARY_DIR}/data/install.sql"
   "${PROJECT_SOURCE_DIR}/data/uninstall.sql"
   DESTINATION "${MRN_DATA_DIR}/")
+if(MSVC AND CMAKE_VERSION VERSION_GREATER "3.1")
+  install(FILES $<TARGET_PDB_FILE:mroonga>
```

Comment:
Where "TARGET_PDB_FILE:mroonga" is defined?
I don't find that definition.

Reply:
@komainu8 

TARGET_PDB_FILE: is CMake builtin feature.
https://cmake.org/cmake/help/v3.1/manual/cmake-generator-expressions.7.html#informational-expressions
So we use `$<TARGET_PDB_FILE:PROJECT_NAME>` is used in here.

Reply:
I see. Thanks!